Practice-Two/src/partials/questions.html

```diff
@@ -0,0 +1,124 @@
+                        </picture>
+                    </div>
+                    <p class="questions-item-text-mobile content-primary">
+                        <span class="questions-item-line"></span>
+                        For any undamaged product, simply return it with its included accessories and packaging along with the original receipt (or gift receipt) within 14 days of the date you receive the product, and we'll exchange it or offer a refund based upon the original payment method.
+                    </p>
+                </article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="content-primary">
+                            Do I need to pay taxes?
+                        </p>
+                        <figure>
+                            <img src="./assets/images/questions/questions-down-arrow.png" alt="questions-down-arrow" />
+                        </figure>
+                    </div>
+                </article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="content-primary">
+                            Can I visit the store or warehouse and order there?
+                        </p>
+                        <figure>
+                            <img src="./assets/images/questions/questions-down-arrow.png" alt="questions-down-arrow" />
+                        </figure>
+                    </div>
+                </article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="content-primary">
+                            How to get a membership card?
+                        </p>
+                        <figure>
+                            <img src="./assets/images/questions/questions-down-arrow.png" alt="questions-down-arrow" />
+                        </figure>
+                    </div>
+                </article>
+            </article>
+            <article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="content-primary">
+                            What is the return policy?
+                        </p>
+                        <figure>
+                            <img src="./assets/images/questions/questions-down-arrow.png" alt="questions-down-arrow" />
+                        </figure>
+                    </div>
+                </article>
+                <article class="questions-item">
+                    <div class="questions-item-title">
+                        <p class="questions-item-description-desktop">How to return products?</p>
+                        <picture>
+                            <source media="(min-width:768px)" srcset="./assets/images/questions/questions-up-arrow.png" />
```

Comment:
Need to explain this

Reply:
Did you mean I have to explain the way I do or is there any problem with this sectioon?